src/common/helpers/github-url.js

```diff
@@ -2,6 +2,12 @@ import parseGitHubUrl from 'parse-github-url';
 
 export const getGitHubRepoUrl = (owner, name) => {
   let repository = !name ? owner : `${owner}/${name}`;
+
+  // if already a valid GH repository
+  if (repository.indexOf('https://github.com/') === 0) {
```

Comment:
Related for example to this:
https://github.com/canonical-ols/build.snapcraft.io/pull/624/files#r110907873